WordPress/Classes/ViewRelated/NUX/LoginViewModel.m

```diff
@@ -187,6 +187,8 @@ - (BOOL)isOnePasswordEnabled
 
 - (void)requestOneTimeCode
 {
+    NSString *message = NSLocalizedString(@"Sending SMS!", @"The app will request One Time Code to be sent via SMS");
```

Comment:
If this is a message to be shown for a duration of time independent of how long it takes to send the SMS, I think it should say "SMS Sent"